atomicfu-gradle-plugin/build.gradle

```diff
@@ -20,15 +20,15 @@ dependencies {
 
     compileOnly gradleApi()
     compileOnly 'org.jetbrains.kotlin:kotlin-stdlib'
```

Comment:
Should this dependency be updated too, or removed (because kotlin-stdlib should be added automatically)?

Reply:
fixed